spec/models/user_spec.rb

```diff
@@ -127,4 +127,15 @@
     @relation = User.reflect_on_association(:events)
     expect(@relation.macro).to eq :has_and_belongs_to_many
   end
+
+  it "has the admin attribute" do
+    user = FactoryBot.build(:user)
+    expect(user).to have_attributes(:admin => false)
+  end
+
+  it "has the admin attribute set to true, if it is an admin" do
```

Comment:
Maybe test for 'has the admin attribute set to false, if it is not an admin' as well?